client/src/helpers/api.js

```diff
@@ -0,0 +1,39 @@
+import axios from 'axios';
+
+const axiosInstance = axios.create({
+  baseURL: 'http://localhost:3300/api',
+  // setting headers here to pass a token to
```

Comment:
great job adding comments to explain a lot of your code. This looks like it will be super helpful if you ever need to come back to it later.